soap_tmpl.go

```diff
@@ -93,6 +93,10 @@ type SOAPClient struct {
 	headers []interface{}
 }
 
+type Empty struct{
```

Comment:
I think it still better to keep original response event if it doesn't contain any response fields, because if at some point you decide to add response data you don't need to change signature so it ensures backwards compatibility.

Reply:
if i define a one-way operation,  gowsdl will generate an blank resp ![image](https://user-images.githubusercontent.com/10744647/50625318-5c399500-0f62-11e9-97a1-cbb826f76c4f.png)

Reply:
Hi @54niyu, I just saw there is separate PR for solving empty responses https://github.com/hooklift/gowsdl/pull/97

Reply:
Hi  @anjmao  ,  I revert the empty response change and make upper type name fixed first.

Reply:
Hi  @anjmao   I revert the empty response change  and  have upper basic type name fixed first .

Reply:
Cool thanks 👍